### DISCUSSION TITLE
希望添加一个监控实时帧率的功能

### DISCUSSION THREAD

**Discussion by snakeqing** [zhongyang219/TrafficMonitor#1081] [Ideas]
希望添加一个监控实时帧率的功能

**Comment by Omoinemie** [comment]
网络工具就专做网络，搞那些乱七八糟的做什么

**Comment by CarterCL** [comment]
希望添加一个监控心率的功能
希望添加一个地震预警的功能
希望添加一个天气预报的功能

**Comment by soloopooo** [comment]
> 希望添加一个监控心率的功能 希望添加一个地震预警的功能 希望添加一个天气预报的功能

别尬黑，有一说一监控心率功能确实可以当做可选插件开发进去，有相关开源软件已经实现了。

可以看下我的 [HeartRate OBS Monitor](https://github.com/soloopooo/HeartRate_Python_OBS)
